ODPi-Runtime.md

```diff
@@ -273,7 +473,7 @@ Best practices for compatible apps to be portable and operator friendly:
 
 -   Applications SHOULD NOT assume that an Oracle JRE is being used.
 
--   Applciations SHOULD include both Microsoft batch or PowerShell as well as Unix-compatible shell scripts.
+-   Applications SHOULD include both Microsoft batch or PowerShell as well as Unix-compatible shell scripts.
```

Comment:
Sure thing - this specific one I've pulled into my PR for ODPI-188 - feel free to revert.

The others you mentioned came from IBM - I'll wait for @lcham to comment.